.github/workflows/cloud-tpu-ci-nightly.yml

```diff
@@ -16,8 +16,8 @@ jobs:
     strategy:
       fail-fast: false  # don't cancel all jobs on failure
       matrix:
-        python-version: ["3.10"]  # TODO(jakevdp): update to 3.11 when available.
-        jaxlib-version: [latest, nightly]
+        python-version: ["3.10"] # TODO(jakevdp): update to 3.11 when available.
```

Comment:
Update to 3.11? It is available now.

Reply:
I'd like to leave that for a separate change, and we should probably do it more holistically throughout our CI. We should arguably also test 3.8 on TPU, since that's the default version that comes on the VM.